client/src/pages/GuidePage/GuidePage.component.tsx

```diff
@@ -82,6 +83,48 @@ export function GuidePage(props: GuidePageProps): JSX.Element {
         <MenuButton start='close' content='text' end='check_circle' size='xl' loading/>
       </Block>
 
+      <Block gap='xs' direction='xy' align='center' padding='xl'>
+        <Format v='uppercase' value='uppercase'/>
+        <Format v='uppercase' placeholder/>
+        <Format v='lowercase' value='LOWERCASE'/>
+        <Format v='lowercase' placeholder/>
+        <Format v='capitalize' value='   capitalize    capitALize'/>
+        <Format v='capitalize' placeholder/>
+        <Format v='title' value='title TITle'/>
+        <Format v='title' placeholder/>
+
+        <Format v='number' value={120202003.12}/>
+        <Format v='number' placeholder/>
+        <Format v='amount' value={12256333.12}/>
+        <Format v='amount' placeholder/>
+        <Format v='percent' value={0.526}/>
+        <Format v='percent' placeholder/>
+        <Format v='currency' value={1025}/>
+        <Format v='currency' placeholder/>
+        <Format v='size' />
+        <Format v='size' value={1024.56}/>
+        <Format v='size' placeholder />
+        <Format v='weight' value='fghj'/>
+        <Format v='weight' value='12335'/>
+        <Format v='weight' placeholder />
+
+        <Format v='day' value='2024-02-27'/>
+        <Format v='day' placeholder/>
```

Comment:
You don't need to place example with placeholder for each variant. Only for type string, date ...